ci-operator/jobs/interconnect-qe/interconnect-tests/interconnect-qe-interconnect-tests-main-periodics.yaml

```diff
@@ -6,17 +6,16 @@ periodics:
   decoration_config:
     skip_cloning: true
   extra_refs:
-  - base_ref: main
```

Comment:
We should never need to manually update a prow job config.

These get automatically generated and updated based on changes we make in our configs. So if the job doesn't have the correct values then we need to run `make update` to fix the values.

I did notice pj-rehearse didn't recognize something and I'm guessing this is why we are attempting to edit the jobs itself?

Reply:
New changes are made in files.